client/src/contexts/FeedbackContext.tsx

```diff
@@ -0,0 +1,46 @@
+import { getFromStorage, setInStorage } from 'API/localStorage';
+import { mockFeedbackData } from 'components/Data/mockFeedbackData';
+import React, { createContext, FC, useState, useCallback, useEffect } from 'react';
+import { Feedback } from 'types';
+
+type FeedbackContextData = {
+  feedback: Feedback[];
+  addFeedback: (newFeedback: Feedback) => void;
+};
+
+const FeedbackContext = createContext<FeedbackContextData>({
+  feedback: [],
+  addFeedback: () => {
+    return null;
+  }
+});
+
+const LOCALSTORAGE_LABEL = 'feedback';
+
+export const FeedbackProvider: FC<React.ReactNode> = ({ children }) => {
+  const [feedback, setFeedback] = useState<Feedback[]>([]);
+
+  const fetchFeedback = useCallback(() => {
+    const feedbackData = getFromStorage(LOCALSTORAGE_LABEL);
```

Comment:
I can't understand what in this variable `LOCALSTORAGE_LABEL`
this name does not mean that it because it is common name. You can have multiple variables and name each `LOCALSTORAGE_LABEL`
I think the best name will be `FEEDBACK_LOCALSTORAGE_LABEL`